ocfweb/docs/docs/services/account/content-removal.md

```diff
@@ -0,0 +1,46 @@
+[[!meta title="Content Removal Policy"]]
+
+
+## Introduction
+
+The OCF maintains a strict policy of not removing or modifying content
+hosted by user accounts unless legally compelled to do so, or if such
+content is in violation of existing OCF account policies.
+
+This means the OCF cannot, and will not, change or remove any text,
+pictures, information, or other content hosted by an OCF user. Only the
+user who owns the relevant OCF account can modify the data we host on
+their behalf. We do however reserve the right to deactivate inactive
+accounts at our discretion.
+
+### Options for removing content
+
+If you would like a document hosted by the OCF to be modified or removed,
+you have a few options.
```

Comment:
Emphasize that this is an exceptional case and should not be motivated by a whim; further, the OCF avoids intervention when possible. Ex:
> If you take issue with the content hosted on the website of another user, the responsibility falls solely on you to resolve any conflict. [...] Only under the circumstances below will the OCF intervene to have content removed without the owner's explicit consent.